ballerina/natives.bal

```diff
@@ -0,0 +1,383 @@
+    name: "initialize"
+} external;
+
+function externCounterRegister(Counter counter) returns error? = @java:Method {
+    'class: "io.ballerina.stdlib.observe.nativeimpl.CounterRegister",
+    name: "register"
+} external;
+
+function externCounterUnRegister(Counter counter) = @java:Method {
+    'class: "io.ballerina.stdlib.observe.nativeimpl.CounterUnregister",
+    name: "unregister"
+} external;
+
+function externCounterIncrement(Counter counter, int amount) = @java:Method {
+    'class: "io.ballerina.stdlib.observe.nativeimpl.CounterIncrement",
+    name: "increment"
+} external;
+
+function externCounterReset(Counter counter) = @java:Method {
+    'class: "io.ballerina.stdlib.observe.nativeimpl.CounterReset",
+    name: "reset"
+} external;
+
+function externCounterGetValue(Counter counter) returns int = @java:Method {
+    'class: "io.ballerina.stdlib.observe.nativeimpl.CounterGetValue",
+    name: "getValue"
+} external;
+
+# This represents the metric type - gauge, that can hold instantaneous, increased or decreased value
+# during the usage.
+#
+# + name - Name of the counter metric
+# + description - Description of the counter metric
+# + metricTags - Tags associated with the counter metric
+# + statisticConfigs - Array of StatisticConfig objects which defines about the statistical calculation
+#                      of the gauge during its usage
+public class Gauge {
+
+    public string name;
+    public string description;
+    public map<string> metricTags;
+    public StatisticConfig[] statisticConfigs;
+
+    # This instantiates the Gauge object. Name field is mandatory, and description, tags, and statitics config fields
+    # are optional and have its own default values when no params are passed.
+    #
+    # + name - Name of the Gauge instance
+    # + desc - Description of the Gauge instance. If no description is provided, the the default empty string
+    #          will be used
+    # + tags - The key/value pair of Tags. If no tags are provided, the default nil value will be used
+    # + statisticConfig - Statistics configurations array is used for statistics calculation. In case if empty
+    #                     statistics configurations array is passed, then statistics calculation will be disabled.
+    #                     If nil () is passed, then default statistics configs will be used for the statitics
+    #                     calculation
+    public function init(string name, string? desc = "", map<string>? tags = (),
+               StatisticConfig[]? statisticConfig = ()) {
+        self.name = name;
+        self.description = desc ?: "";
+        self.metricTags = tags ?: DEFAULT_TAGS;
+        self.statisticConfigs = statisticConfig ?: DEFAULT_GAUGE_STATS_CONFIG;
+        externGaugeInit(self);
+    }
+
+    //# Performs the necessary native operations during the initialization of the gauge.
+    //function initialize() = external;
```

Comment:
Let's remove this as well. Same as above.

Reply:
Removed in https://github.com/ballerina-platform/module-ballerina-observe/pull/1/commits/ed8df9f5f8ca5121b0dd0a3a15cb43f5ed96749a